docs/self-managed/components/orchestration-cluster/core-settings/configuration/configuration-mapping.md

```diff
@@ -0,0 +1,94 @@
+| <span className="badge badge--1-to-1">1-to-1</span>                                                                                                           | <p>New Camunda 8.8 property with an existing equivalent legacy property or set of properties.</p><p>Backwards compatibility is supported as follows:</p><p><ul><li><p>If you have defined the new property, it is used.</p></li><li><p>If you have not defined the new property, the legacy property is used (if you have defined it).</p></li></ul></p>                                                                                                                                                                                                  |
+| <span className="badge badge--breaking-change-double-configuration-" style={{ width: '150px', fontSize: '75%'}}>Breaking change (double configuration)</span> | <p>New Camunda 8.8 property with an existing equivalent legacy property or set of properties.</p><p>Backwards compatibility is **not** supported.</p><p><ul><li><p>Applies if a legacy configuration is defined with values that match the new configuration default or custom values.</p></li><li><p>If the legacy configuration value and the new configuration value match, the legacy property is accepted, even if not supported.</p></li><li><p>If the values do not match, the application will not start and an error is shown.</p></li></ul></p> |
+
+### Recommended actions
+
+**As part of upgrading to 8.8, Camunda recommends you replace any legacy properties shown in this table with the equivalent new property.**
+
+You can define configuration properties as environment variables using [Spring Boot conventions](https://docs.spring.io/spring-boot/reference/features/external-config.html#features.external-config.typesafe-configuration-properties.relaxed-binding.environment-variables). To define an environment variable, convert the configuration property to uppercase, remove any dashes, and replace any delimiters (.) with \_. For example:
+
+| Property                                   | Environment variable                    |
+| :----------------------------------------- | :-------------------------------------- |
+| `camunda.api.grpc.address`                 | `CAMUNDA_API_GRPC_ADDRESS`              |
+| `camunda.api.grpc.min-keep-alive-interval` | `CAMUNDA_API_GRPC_MINKEEPALIVEINTERVAL` |
+
+### Example
+
+In this example, an application uses the following legacy configuration:
+
+```
+camunda.database.url=http://prod-db.com:54321
+camunda.operate.opensearch.url=http://prod-db.com:54321
+camunda.tasklist.opensearch.url=http://prod-db.com:54321
+```
+
+#### Option 1: Strict migration
+
+The recommended strict migration approach is to remove the legacy properties and add the corresponding new property:
+
+```
+camunda.data.secondary-storage.opensearch.url=http://prod-db.com:54321
+```
+
+#### Option 2: Double configuration
```

Comment:
Lets remove this Option. I don't believe there is a use case for having it.

Reply:
And as mentioned above, let's remove double configuration mentions. Customers just need to upgrade them, and we mentioned that. Old properties could be kept as a note.